simulation/Scintillator_Energy/example/energy_simulation_example1.py

```diff
@@ -0,0 +1,223 @@
+import numpy as np
+import matplotlib.pyplot as plt
+
+#Define your parameters:
+
+#Length of orthogonal scintillators
+L=0.3
+
+#This is the maximum detectable angle in degrees,leads to the height between scint. layers by h=L/tan(θ_max)
+theta_max=60 
+
+#Minimum and maximum detectable muons energy(in GeV):
+E_min=1
+E_max=50
+
+#Number of possible energy values(this depends on SiPM's accuracy):
+N_E=6
+
+#For a random sample define 1,for a perfect CM distribution 0:
+random=1
+
+def MetropolisHastings(pdf, N, x_init, sigma):
+    chain_points=np.zeros((N,x_init.shape[0]))
+    stepsize=sigma
+    x_old=x_init
+    for i in range(N):
+      x_new =np.random.uniform(low=x_old-(1/2)*stepsize,high=x_old+(1/2)*stepsize,size=x_old.shape)
+      p_accept=min(1, pdf(x_new)/pdf(x_old))
+      accept = (np.random.random()<p_accept)
+      if accept:
+        x_old=x_new
+      chain_points[i,:]=x_old
+
+    return chain_points
+    
+def pdf_theta(x,theta_max=theta_max*np.pi/180):
+    if (x>theta_max) or (x<0):
+        return 0
+    return np.cos(x)**2
+
+def norm(theta,E_min=E_min,E_max=E_max,N_E=N_E):
+    dE=(E_max-E_min)/(N_E-1)
+    e_k=115
+    e_p=850
+    E=E_min
+    sum=0
+    for i in range (N_E):
+        sum+=0.14*dE*(E**(-2.7))*(((1/(1+1.1*E*np.cos(theta)/e_p))+(0.054/(1+1.1*E*np.cos(theta)/e_k))))
```

Comment:
where do the values come from? -fh

Reply:
I will add some theoretical stuff on slack on the theoretical stuff and formula's.I can add links for those in the readme